cli/batch-score.sh

```diff
@@ -8,34 +8,42 @@ export ENDPOINT_NAME="<YOUR_ENDPOINT_NAME>"
 
 export ENDPOINT_NAME=endpt-`echo $RANDOM`
 
+echo "Creating compute"
 # <create_compute>
 az ml compute create -n batch-cluster --type amlcompute --min-instances 0 --max-instances 5
 # </create_compute>
 
+echo "Creating batch endpoint %ENDPOINT_NAME"
 # <create_batch_endpoint>
 az ml batch-endpoint create --name $ENDPOINT_NAME
 # </create_batch_endpoint>
 
+echo "Creating batch deployment nonmlflowdp for endpoint $ENDPOINT_NAME"
 # <create_batch_deployment_set_default>
 az ml batch-deployment create --name nonmlflowdp --endpoint-name $ENDPOINT_NAME --file endpoints/batch/nonmlflow-deployment.yml --set-default
 # </create_batch_deployment_set_default>
 
+echo "Showing details of the batch endpoint"
 # <check_batch_endpooint_detail>
 az ml batch-endpoint show --name $ENDPOINT_NAME
 # </check_batch_endpooint_detail>
 
+echo "Showing details of the batch deployment"
 # <check_batch_deployment_detail>
 az ml batch-deployment show --name nonmlflowdp --endpoint-name $ENDPOINT_NAME
 # </check_batch_deployment_detail>
 
+echo "Invoking batch endpoint with public URI (MNIST)"
 # <start_batch_scoring_job>
 JOB_NAME=$(az ml batch-endpoint invoke --name $ENDPOINT_NAME --input-path folder:https://pipelinedata.blob.core.windows.net/sampledata/mnist --query name -o tsv)
```

Comment:
What I meant by this comment on the docs PR: https://github.com/MicrosoftDocs/azure-docs-pr/pull/198470#discussion_r879959605

Is that should this command on line 38 still be using --input-path or should it be using --input and --input-type?